homeassistant/components/elkm1/__init__.py

```diff
@@ -284,7 +288,7 @@ def _element_changed(element, changeset):
     except asyncio.TimeoutError as exc:
         raise ConfigEntryNotReady(f"Timed out connecting to {conf[CONF_HOST]}") from exc
 
-    elk_temp_unit = elk.panel.temperature_units  # pylint: disable=no-member
+    elk_temp_unit = getattr(elk, "panel").temperature_units
```

Comment:
same as above?

Reply:
keypads is a dynamically created attribute. Typing complains. The other option is to cast (AFAIK, I’m not yet a typing expert)